assets/agenda/utils.ts

```diff
@@ -1015,42 +1023,54 @@ export const getCoverageTooltip = (coverage: any, beingUpdated?: any) => {
     const slugline = coverage.item_slugline || coverage.slugline;
     const coverageType = getCoverageDisplayName(coverage.coverage_type);
     const coverageScheduled = moment(coverage.scheduled);
+    const assigneeText = getCoverageAsigneeName(coverage) ? `, assignee: ${getCoverageAsigneeName(coverage)}` : '';
```

Comment:
Doing it this way would not flag `assignee: ` nor `desk: ` for translation. Consider a better way to append the desk/assignee details onto the tooltip

Reply:
you mean like this :
```.ts
let additionalDetails = '';

    if (assignee) {
        additionalDetails += `, ${gettext('assignee')}: ${assignee}`;
    }

    if (desk) {
        additionalDetails += `, ${gettext('desk')}: ${desk}`;
    }

    if (coverage.workflow_status === WORKFLOW_STATUS.DRAFT) {
        return gettext('{{ type }} coverage {{ slugline }} {{ status_text }}{{additionalDetails}}', {
            type: coverageType,
            slugline: slugline,
            status_text: getCoverageStatusText(coverage),
            additionalDetails: additionalDetails,
        });